core/src/test/java/org/apache/iceberg/rest/RESTCatalogServlet.java

```diff
@@ -20,6 +20,9 @@
 
 import static java.lang.String.format;
 
+import jakarta.servlet.http.HttpServlet;
```

Comment:
fyi I tried to do the same for the `S3SignerServlet` in `iceberg-aws` but that isn't currently working because `iceberg-aws` pulls in https://github.com/apache/iceberg/blob/b0cf9988c9d837c03aacc4fe9d45058b63fb68fd/build.gradle#L505 which in turn pulls in a different Jetty version. Updating the s3mock isn't possible atm because a newer version depends on JDK17